src/bindings/interface/entity.i

```diff
@@ -23,13 +23,13 @@ unset).";
 "Returns the parent of the CellML Entity (or `None` if not set).";
 
 %feature("docstring") libcellml::Entity::hasParent
-"Tests if this entity have the given entity as a parent.";
+"Tests if this entity has a parent returns true if it has a parent and false otherwise.";
```

Comment:
"Tests if this entity has a parent.  Returns true if it has a parent and false otherwise."